lldb/source/Target/Target.cpp

```diff
@@ -335,6 +335,48 @@ BreakpointSP Target::GetBreakpointByID(break_id_t break_id) {
   return bp_sp;
 }
 
+lldb::BreakpointSP lldb_private::Target::CreateBreakpointAtUserEntry() {
+  TargetSP target_sp = shared_from_this();
```

Comment:
Why is this necessary?

Reply:
I removed that line and the use of `target_sp`, hence, these lines also changed:

```cpp
ModuleSP main_module_sp = GetExecutableModule();
BreakpointSP bp_sp = CreateBreakpoint(...)
```

It still works normally and pass the test. Thank you.